content/modules/rest/pages/metadata-api.adoc

```diff
@@ -0,0 +1,202 @@
+= Metadata API
+
+It is possible to introspect the entity structure and relationships between entities in Jmix via the xref:data-model:metadata.adoc[Metadata] mechanism. You can also access these capabilities through the REST API with the different metadata endpoints starting with `/metadata`.
+
+The Metadata API consists of the following endpoints:
+
+
+* <<Entity Metadata>> Information about the entity and detailed information about the entity attributes.
+* <<Fetch Plan Information>> Information about available Fetch Plans for entities and their attributes.
+* <<Enum Values>> Information about available Enums, and their possible values.
```

Comment:
\<\<Enum Values\>\> information about available Enums and their possible values.